roles/tendrl-storage-node/tasks/main.yml

```diff
@@ -22,3 +22,4 @@
     state=latest
 
 - include: tendrl-node-agent.yml
+- include: rsyslog.yml
```

Comment:
It may be better to include `rsyslog.yml` before including `tendrl-node-agent.yml`. That said, this have no impact on triggering rsyslogd restart via notification.

Reply:
+1